src/components/molecules/Button/Button.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, PropsWithChildren, SVGProps, FunctionComponent } from 'react'
```

Comment:
**[blocker]**

FC and FunctionComponent are the same thing. FC is just a shorthand for it, so you won't need both. In the rest of the project FC is used so let's stick with FC for consistenct